bin/include/jvmdefaults.bat

```diff
@@ -24,74 +24,15 @@ set value=""
 :: Second argument is the current value of the jvm options
 :: Third value is the name of the environment variable that jvm options should be set to
 if %java_version% == 8 (
-    set value= ^
-    -XX:+AggressiveOpts ^
-    %current_value%
+    set value=-XX:+AggressiveOpts %current_value%
 )
 
 if %java_version% GEQ 9 if %java_version% LSS 11 (
-    set value= ^
-    -XX:+AggressiveOpts ^
-    --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED ^
-    --add-exports=java.base/sun.nio.ch=ALL-UNNAMED ^
-    --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED ^
-    --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED ^
-    --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED ^
-    --illegal-access=permit ^
-    --add-modules=java.xml.bind ^
-    %current_value%
+    set value=-XX:+AggressiveOpts --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED --illegal-access=permit --add-modules=java.xml.bind %current_value%
```

Comment:
It looks like, you don't use this script to define jvm options for the control utility. So, what is the reason for this change?